management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java

```diff
@@ -65,10 +65,10 @@ public Capability getCapability() {
   @Override
   public Map<String, Statistic<?, ?>> collectStatistics(Context context, Collection<String> statisticNames, long since) {
     Map<String, Statistic<?, ?>> statistics = new HashMap<String, Statistic<?, ?>>(statisticNames.size());
-    EhcacheStatistics ehcacheStatistics = (EhcacheStatistics) findExposedObject(context);
+    StandardEhcacheStatistics ehcacheStatistics = (StandardEhcacheStatistics) findExposedObject(context);
     if (ehcacheStatistics != null) {
       for (String statisticName : statisticNames) {
-        statistics.putAll(ehcacheStatistics.queryStatistic(statisticName, since));
+        statistics.put(statisticName, ehcacheStatistics.queryStatistic(statisticName, since));
```

Comment:
Does it mean that there is not anymore the concept of "groups" of statistics ? 
Before, we had some group name, and when we asked for i.e. `AllCacheGet`, then it would return all associated statistics (AllCacheGetCount, AllCacheGetRate, AllCacheGetLatencyMinimum, ...).
Hence the `.putAll()` ;-)